src/dmd/mars.d

```diff
@@ -692,8 +691,7 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
             // write the output to $(filename).cg
             auto cgFilename = FileName.addExt(mod.srcfile.toString(), "cg");
             auto cgFile = File(cgFilename);
-            cgFile.setbuffer(buf.data, buf.offset);
-            cgFile._ref = 1;
+            cgFile.hijackBufferFrom(buf);
```

Comment:
Breaking change; `cgFile` takes ownership, but just for convenience, as `buf` is destructed right after `cgFile` anyway.